### ISSUE TITLE
Double with multiple: true doesn't seem to work

### ISSUE THREAD

**Issue by rcannood** [viash-io/viash#235]
Config:
```yaml
functionality:
  name: foo
  arguments:
    - type: double
      name: --foo
      multiple: true
      multiple_sep: ";"
  resources:
    - type: python_script
      path: script.py
      text: print(par)
```

Error log:
```bash
$ viash run config2.vsh.yaml -- --foo 0.1 --foo 0.9
[error] --foo has to be a double. Use --help to get more information on the parameters.
Files and logs are stored at '/home/rcannood/workspace/viash_temp/viash_foo731907709239152362'

$ viash run config2.vsh.yaml -- --foo 0.1 
{'foo': [0.1]}

```

It seems it's not looking at the `multiple_sep`, because the following works:
```bash
$ viash run config2.vsh.yaml \
  -c ".functionality.arguments[.type == 'double'].multiple_sep := ':'" -- \
  --foo 0.1 --foo 0.9
{'foo': [0.1, 0.9]}
```

**Comment by rcannood** [comment]
I see what the problem is; the IFS is not quoted:


https://github.com/viash-io/viash/blob/391feeaf67407e5d143eb37ec8d02551371d03d6/src/main/scala/io/viash/wrapper/BashWrapper.scala#L478-L488

**Comment by rcannood** [comment]
While we're looking at BashWrapper, I think the title `# check whether parameters values are of the right type` should be adapted, depending on which test is being performed. Also, a part of this code generates a lot of newlines:
```bash
# check whether parameters values are of the right type














# detect volumes from file arguments
```

(Generated from testbash)

**Comment by rcannood** [comment]
Solved in #237 !
